### ISSUE TITLE
onClick handler for React Swipe does not work when there's exactly 2 swipe items

### ISSUE THREAD

**Issue by thunghan** [lyfeyaj/swipe#65]
Version 2.2.4
1. Open the `examples/react` folder and replace `app/app.jsx` with the following code:
```
import React from 'react';
import ReactDom from 'react-dom';
import { Swipe, SwipeItem } from 'swipejs/react';

import 'swipejs/style.css';
import './style.css';

class ReactSwipeExampleApp extends React.Component {
  constructor(...args) {
    super(...args);
    this.state = {};
    this.handleClick = this.handleClick.bind(this);
  }

  handleClick(n) {
    console.log(`clicked ${n}`)
  }

  render() {
    return (
      <div className='react-swipe-example'>
        <h2>React Swipe Example</h2>
        <Swipe className='custom-swipe-container-class'
               ref='swipe'
               startSlide={0}
               speed={300}
               draggable={true}
               continuous={true}
               disableScroll={false}
               stopPropagation={false}>
          <SwipeItem className='custom-swipe-item-class'
                     onClick={() => this.handleClick(1)}>
            Slide One
          </SwipeItem>
          <SwipeItem className='custom-swipe-item-class'
                     onClick={() => this.handleClick(2)}>
            Slide Two
          </SwipeItem>
        </Swipe>
      </div>
    );
  }
}

ReactDom.render(<ReactSwipeExampleApp />, document.getElementById('react-swipe-example-app'));

```
2. Run the example in the browser
3. Click on the text when the example first loads. The click event should be logged in the console.
4. Now, swipe left (i.e. click and move cursor to the right) by one screen and click the text again. The click event is not logged in the console.

**Comment by lyfeyaj** [comment]
Fixed by #66.

Try version `swipejs@2.2.5`

**Comment by hardfist** [comment]
'width' and 'left' is missing in the cloned children's inline style, which cause problem. @lyfeyaj
